pkg/autopilot/checks/removedapis.go

```diff
@@ -43,7 +43,7 @@ func removedInVersion(candidate schema.GroupVersionKind) string {
 }
 
 // Sorted array of removed APIs.
-var removedGVKs = [59]removedAPI{
+var removedGVKs = [65]removedAPI{
```

Comment:
That's the way to tell the compiler to figure it out on its own:

```suggestion
var removedGVKs = [...]removedAPI{
```